libraries/api-extractor/src/test/DocItemLoader.test.ts

```diff
@@ -19,6 +19,11 @@ function testErrorHandler(message: string, fileName: string, lineNumber: number)
   capturedErrors.push({ message, fileName, lineNumber });
 }
 
+function assertCapturedErrors(expectedMessages: string[]): void {
+  assert.deepEqual(capturedErrors.map(x => x.message), expectedMessages,
```

Comment:
Is it an expectation that the errors need to be in this exact same order?
Otherwise check that the length matches, and that all strings are present as capturedError messages